packages/core/admin/ee/server/services/audit-logs.js

```diff
@@ -73,6 +74,7 @@ const createAuditLogsService = (strapi) => {
     async register() {
       this._provider = await localProvider.register({ strapi });
       this._eventHubUnsubscribe = strapi.eventHub.subscribe(handleEvent.bind(this));
+      this._deleteExpiredJob = scheduleJob('0 0 * * *', () => this._provider.deleteExpiredEvents());
```

Comment:
I think you can pass the function directly: `this._provider.deleteExpiredEvents` instead of `() => this._provider.deleteExpiredEvents()`